Plugins/SafeDIGenerateDependencyTree/SafeDIGenerateDependencyTree.swift

```diff
@@ -112,3 +133,17 @@ extension SafeDIGenerateDependencyTree: XcodeBuildToolPlugin {
     }
 }
 #endif
+
+extension Data {
+    fileprivate func write(toPath filePath: String) throws {
+#if os(Linux)
+        try write(to: URL(fileURLWithPath: filePath))
+#else
+        if #available(macOS 13.0, iOS 16.0, tvOS 16.0, watchOS 9.0, *) {
+            try write(to: URL(filePath: filePath))
+        } else {
+            try write(to: URL(fileURLWithPath: filePath))
+        }
+#endif
+    }
```

Comment:
This can't be simplified?
```
         if #available(macOS 13.0, iOS 16.0, tvOS 16.0, watchOS 9.0, *) {
             try write(to: URL(filePath: filePath))
         } else {
             try write(to: URL(fileURLWithPath: filePath))
         }
```

Reply:
Ah nvm, it's `#if something` not `if #something`

Reply:
yup. tried that 🙂. Took me awhile to get this right. Linux availability support is... bad (read: nonexistent).